code/parachain/runtime/composable/src/lib.rs

```diff
@@ -606,6 +640,12 @@ construct_runtime!(
 		Assets: pallet_assets = 57,
 		AssetsRegistry: assets_registry = 59,
 
+		Referenda: pallet_referenda = 76,
+		ConvictionVoting: pallet_conviction_voting = 77,
+		OpenGovBalances: balances::<Instance2> = 78,
```

Comment:
i am ok, but as i mentioned, it may have no so great integration with ui as assets pallet wrapper with custom token mint.